org.opent2t.sample.hub.superpopular/com.nest.hub/js/thingTranslator.js

```diff
@@ -37,8 +37,8 @@ class Translator {
      * Get the list of devices discovered through the hub.
      */
     getPlatforms(expand, payload) {
-        if(payload !== undefined){
-            return this._providerSchemaToPlatformSchema( payload, expand );
+        if (payload !== undefined) {
```

Comment:
minor. switch conditions for readability

Reply:
Fixed.